src/spikeinterface/core/sparsity.py

```diff
@@ -353,41 +417,47 @@ def from_snr(cls, templates_or_sorting_analyzer, threshold, noise_levels=None, p
         return cls(mask, unit_ids, channel_ids)
 
     @classmethod
-    def from_ptp(cls, templates_or_sorting_analyzer, threshold, noise_levels=None):
+    def from_ptp(cls, templates_or_sorting_analyzer, threshold):
         """
         Construct sparsity from a thresholds based on template peak-to-peak values.
-        Use the "threshold" argument to specify the SNR threshold.
+        Use the "threshold" argument to specify the peak-to-peak threshold.
+
+        Parameters
+        ----------
+        templates_or_sorting_analyzer : Templates | SortingAnalyzer
+            A Templates or a SortingAnalyzer object.
+        threshold : float
```

Comment:
I would use threshold_uv if now it is always absolut to sure we avoid mistake

Reply:
well that depends on whether return scaled is true. Shoudl we throw an error if it's not so that this only works with scaled templates?